data/nlu.yml

```diff
@@ -90,3 +90,18 @@ nlu:
     - are you a human?
     - am I talking to a bot?
     - am I talking to a human?
+
+
+- intent: hello_world
```

Comment:
the `hello_world` example was part of the tutorial. do you want it to be a part of your bot's conversation flow?